### DISCUSSION TITLE
Make format option to omit newline before opening braces

### DISCUSSION THREAD

**Discussion by MadProbe** [dnSpyEx/dnSpy#178] [Ideas]
Omitting newline before opening brace in functions, if, for, while statements looks more pleasant and rational for me, as well as it increases code density so it can fit much more code lines in one screen, especially in conjunction with existing `Compress lines that don't have alphanumeric characters` option

**Comment by ElektroKill** [comment]
Duplicate of #2. The ability to customize the styling of the output C# code is a planned feature for the 7.0 major release together with an update to the ILSpy decompiler engine.
